applications/app/views/fragments/mediaBody.scala.html

```diff
@@ -65,6 +66,14 @@
                                             <meta itemprop="datePublished" content="@video.trail.webPublicationDate.toString("yyyy-MM-dd")" />
                                             <meta itemprop="name" content="@video.metadata.webTitle" />
                                             <meta itemprop="uploadDate" content="@video.trail.webPublicationDate.toString("yyyy-MM-dd")" />
+
+                                            @video.mediaAtom.map { ma =>
+                                                @ma.duration.map { duration =>
+                                                    <meta itemprop="duration" content="@MediaAtom.isoDuration(duration)" />
+                                                }
+                                            }
+                                            @* TODO: Set meta thumbnail once we have better image data in model *@
```

Comment:
Got it. However what guarantee do I have this TODO will not still be here in 6months? 😀